ExampleApp/ExampleApp/Screens/CentralServices/CentralServicesViewController.swift

```diff
@@ -0,0 +1,94 @@
+import RxBluetoothKit
+import RxSwift
+import UIKit
+
+class CentralSericesViewController: UITableViewController {
+
+    init(peripheral: Peripheral, bluetoothProvider: BluetoothProvider) {
+        self.peripheral = peripheral
+        self.bluetoothProvider = bluetoothProvider
+        super.init(nibName: nil, bundle: nil)
+
+        navigationItem.title = "Peripheral's services"
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupBindings()
+        tableView.register(CentralServiceCell.self, forCellReuseIdentifier: CentralServiceCell.reuseId)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(true)
+        didAppearSubject.onNext(())
+    }
+
+    // MARK: - TableView
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        services.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: CentralServiceCell.reuseId, for: indexPath) as? CentralServiceCell else {
+            fatalError("Something went wrong :(")
+        }
+
+        let service = services[indexPath.row]
+        cell.uuidLabel.text = service.uuid.uuidString
+        cell.isPrimaryLabel.text = "isPrimary: \(service.isPrimary)"
+        cell.characterisicsCountLabel.text = "chcarac. count: \(service.characteristics?.count ?? -1)"
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let service = services[indexPath.row]
+
+        bluetoothProvider.characteristics(for: service)
+            .subscribe(
+                onNext: { [weak self] in self?.pushCharacteristicsController(with: $0) },
```

Comment:
Discovering characteristics could take some time, so I think it is possible to perform another action during this time. Do you think it makes sense to block the UI in some way during this operation?

Reply:
I would say it exceeds the scope of this example app, what do you think?

Reply:
Ok. However, can you add a comment informing about this, so when someone will use this code, he will be aware of this  limitation.

Reply:
Good idea. Added a comment! :)